mobile/src/screens/survey/survey.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Link, SignBackground, Text } from '#libs/components/components';
+import { RootScreenName } from '#libs/enums/enums';
+import { useAppDispatch, useAppSelector, useCallback } from '#libs/hooks/hooks';
+import { actions as authActions } from '#slices/auth/auth';
+
+import { PreferencesStep } from './components/components';
+import { styles } from './styles';
+
+const Survey: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { userId } = useAppSelector(({ auth }) => ({
+    userId: auth.authenticatedUser?.id,
```

Comment:
Is user already logged for Survey screen, if so then remove optional chaining

Reply:
@v0ldemar01 i get a ts error if i remove it, that authenticatedUser is possibly null

Reply:
@iamAlinaaa You should use `(auth.authenticatedUser as UserAuthResponseDto).id`
Here we are sure that we always have a user, so we can cast type